bigchaindb/core.py

```diff
@@ -274,8 +277,27 @@ def get_status(self, txid):
             or 'backlog'). If no transaction with that `txid` was found it
             returns `None`
         """
-        _, status = self.get_transaction(txid, include_status=True)
-        return status
+        try:
+            statuses = self.get_blocks_status_containing_tx(txid).values()
```

Comment:
Why does this not just call `Bigchain.get_transaction(.., include_status=True)` and use that?